ext/opcache/jit/zend_jit.c

```diff
@@ -3970,7 +3970,11 @@ static int zend_jit(const zend_op_array *op_array, zend_ssa *ssa, const zend_op
 				case ZEND_OP_DATA:
 				case ZEND_SWITCH_LONG:
 				case ZEND_SWITCH_STRING:
+					break;
 				case ZEND_MATCH:
+					if (zend_jit_tail_handler(&dasm_state, opline)) {
+						goto jit_failure;
+					}
```

Comment:
This way we will generate dead code after the jump to MATCH handler.
Anyway, this is better than crash and should be enough for PHP-8.2 and PHP-8.3. 
Please add some comments, something like - "We have to exit to VM, because MATCH handler performs N-may jump and we can't generate a simple (``opcache.jit=1201``)  JIT code"